pkg/client/unversioned/clientcmd/client_config.go

```diff
@@ -358,21 +358,21 @@ func (inClusterClientConfig) ClientConfig() (*restclient.Config, error) {
 	return restclient.InClusterConfig()
 }
 
-func (inClusterClientConfig) Namespace() (string, error) {
```

Comment:
Add a couple type assertions.  this dropped out of the `ClientConfig` interface at some point.